Kitsune/ClubPenguin/Login.php

```diff
@@ -27,14 +27,15 @@ protected function handleLogin($socket) {
 		$password = Packet::$Data['body']['login']['pword'];
 		
 		if($penguin->database->usernameExists($username) === false) {
+			password_verify('KitsuneTimingAttack', '$2y$12$5hf7vOtBjsNJf6oZSsnyXOlcLTRblTwSg550SR0ohxVUkItmXhMM6'); //Mitigate timing attacks
```

Comment:
I can see how semantically that would violate the contract of a login server, so I think it's a good consideration, but I don't see how it's a security risk. Thank you for taking time to explain this to me because prior I didn't know about timing attacks.